Samples/V1.0Samples/LocalMediaSamples/AudioVideoPlaybackBot/ServiceConfiguration.Cloud.cscfg

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ServiceConfiguration serviceName="AudioVideoPlaybackBot" xmlns="http://schemas.microsoft.com/ServiceHosting/2008/10/ServiceConfiguration" osFamily="5" osVersion="*" schemaVersion="2015-04.2.6">
+<ServiceConfiguration serviceName="AudioVideoPlaybackBot" xmlns="http://schemas.microsoft.com/ServiceHosting/2008/10/ServiceConfiguration" osFamily="6" osVersion="*" schemaVersion="2015-04.2.6">
```

Comment:
I am not clear why this is needed.

The requirements for this sample is .NetFramework 4.6.2.  [OsFamily 5](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-guestos-update-matrix#family-5-releases
) should cover this.

Further, this should have the same effect when running locally vs running on Azure, and these have been validated locally.

Reply:
I wish I could help out. It was just my experience that when I cloned the repo as is and followed the sample instructions to deploy, the role instance wouldn't even start because of the exception in #68 . But when I changed it to osFamily 6, it worked as expected.

Reply:
Can you try adding a nuget reference to [System.ValueTuple (>= 4.3.0)](https://www.nuget.org/packages/System.ValueTuple/4.3.0) instead?

Updating OsFamily may have other implications (on the media side), and would require further testing.

Reply:
My colleague had tried adding the nuget reference but that didn't seem to work. He then manually installed .Net Framework 4.7.2 into the osFamily 5 role instance and that worked. Which is what prompted us to use osFamily 6 which also worked. 

With the changes in this PR, our testing has worked fine so I'll let this in your court to do further testing. In our experience though, the sample as is before the PR does not work.